hash/nes.xml

```diff
@@ -61818,6 +61818,22 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 		</part>
 	</software>
 
+<!-- Built-in level select at title screen. Press in sequence: A, B, B, followed by A 1 to 5 times for stages 1-1, 2-1, 3-1, 4-1, and 5-1, respectively. -->
```

Comment:
Should this be in a usage note?  See how software lists use `<info name="usage" value="…" />` for usage notes supported by the internal UI.

Reply:
Sounds good. Embarrassing but...I didn't actually know where the heck usages popped up until just now, since I don't use much other than the CLI. I do look in the TAB menu though. There's not a way to view them there, no?

Reply:
I don’t know if you can see them while emulation is running.  They’re often helpful instructions for launching the software (more important for computer-like systems), so it makes sense to have them visible in the software selection menu.